sympy/vector/coordsysrect.py

```diff
@@ -98,6 +213,65 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        if transformation is None:
+            if parent is None:
```

Comment:
It could be, but the one difference between these two cases is with `variables`. When we have `parent` we can construct transformation equations using base scalars from `parent` system. Otherwise we can't.